.gitignore

```diff
@@ -28,6 +28,7 @@ test-results*.xml
 /client/chart.json
 /client/style.json
 *xunit_*.xml
+/results
```

Comment:
Ignoring the result dir.

We're going with the same naming choice as in the Gutenberg repo.